src/storage/sftp/storage.c

```diff
@@ -1343,7 +1343,8 @@ storageSftpNew(
                     "HINT: libssh2 compiled against non-openssl libraries requires --repo-sftp-private-key-file and"
                     " --repo-sftp-public-key-file to be provided\n"
                     "HINT: libssh2 versions before 1.9.0 expect a PEM format keypair, try ssh-keygen -m PEM -t rsa -P \"\" to"
-                    " generate the keypair"));
+                    " generate the keypair\n"
+                    "HINT: You may also want to check /var/log/auth.log of the SFTP server"));
```

Comment:
I'll defer to @jreidthompson on this, but I wonder if this message is a bit too specific RE the log file name. Maybe something like:
```
Check the authorization log on the SFTP server.
```
On the other hand, water it down this much I wonder if it is very useful.

Reply:
I agree that /var/log/auth.log is too specific, the log name and location can vary by OS, sftp server, or configuration, etc. I do think it would be useful. A quick check indicates that a google search for "$OS ssh authorization log file" or  "$DISTRO ssh authorization log" provides decent references for persons that may not be familiar with where logging is directed.